lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -903,6 +903,11 @@ def get_proposed(self):
             existing = ref[k].get('existing', ref[k]['default'])
             if playval == existing and ref['_state'] == 'present':
                 continue
+            if isinstance(ref[k]['default'], dict):
```

Comment:
@chrisvanheuveln review comment: Need to modify this to use `existing.values` instead of `ref[k]['default']` and combine both `if isinstance` and `if not all` checks.

Reply:
Good find!